js/platform/main/src/components/ImportData/ImportDataModal.js

```diff
@@ -172,6 +177,45 @@ class ImportDataModal extends Component {
         }
     };
 
+    importJSON = async file => {
```

Comment:
@buralog This method should be in FHIRService. You might add an import method with a file type.